### ISSUE TITLE
Apostrophes in template names

### ISSUE THREAD

**Issue by dscorbett** [portstrom/parse_wiki_text#4]
Repeated apostrophes in template names, as in `{{''}}`, do not represent formatting and should be parsed as text. I suspect the same is true of template parameter names, though I haven’t checked. Compare `[['']]`, where they also are not formatting, which this crate parses correctly.

**Comment by Erutuon** [comment]
Correct, in template names and in template parameters and values as well. `'''` doesn't have any significance there as well. As far as I can tell, italics and bolding syntax is only significant in the very last stage of parsing, after templates have been expanded, when HTML is generated.
